coreos-ostree-importer/README.md

```diff
@@ -99,3 +99,24 @@ You'll have to update the body with new information you'd like to use. Then run:
 ```
 ./publisher.py
 ```
+
+## Fedora Messaging sender from the Fedora CoreOS Pipeline
+
+Included in this directory is a file (`fcos-pipeline-ostree-import-request.py`)
```

Comment:
Minor: maybe something a little more concise, e.g. `send-ostree-import-request.py`? The fact that it's used in the FCOS pipeline is a detail I think.